dockerpty/pty.py

```diff
@@ -249,3 +249,103 @@ def _hijack_tty(self, pumps):
                 except SSLError as e:
                     if 'The operation did not complete' not in e.strerror:
                         raise e
+
+
+class ExecPTY(PseudoTerminal):
+    def __init__(self, client, container, interactive=True, stdout=None, stderr=None, stdin=None):
+        """
+        Initialize the PTY using the docker.Client instance and container dict.
+        """
+        super(ExecPTY, self).__init__(client, container, interactive=interactive,
+                                      stdout=stdout, stderr=stderr, stdin=stdin)
+
+        self.exec_id = None
```

Comment:
I'd like to avoid the inheritance if possible.  Maybe a bunch of this can be added to the `PseudoTerminal` class directly?  At least the args are the same.